File: 1351-count-negative-numbers-in-a-sorted-matrix/1351-count-negative-numbers-in-a-sorted-matrix.java
class Solution {
    public int countNegatives(int[][] grid) {
        int ans = 0;
        for(int[] mat : grid){
            // int count = 0;
            // boolean flag = false;
            for(int i=0;i<mat.length;i++){
                if(mat[i] < 0){
                    
                    ans += mat.length - i;
                    break;
                }
                // if(flag)
                    
            }
            
        }
        return ans;
    }
}

File: 1351-count-negative-numbers-in-a-sorted-matrix/1351-count-negative-numbers-in-a-sorted-matrix.js
/**
 * @param {number[][]} grid
 * @return {number}
 */
var countNegatives = function(grid) {
    let count = 0;
    for(mat of grid){
        for( let i =0; i< mat.length; i++){
            if(mat[i] < 0){
                count += mat.length - i;
                break
            }
        }
    }
    return count;
};